src/Donations/Endpoints/ListDonations.php

```diff
@@ -87,9 +101,20 @@ public function handleRequest(WP_REST_Request $request)
             $data[] = DonationResponseData::fromObject($donation)->toArray();
         }
 
+        /**
+         * @var DonationsListTable $listTable
+         */
+        $listTable = give(DonationsListTable::class);
+        $listTable->items($data);
+
+        // Sort
+        if ($sortColumn = $request->get_param('sortColumn')) {
+            $listTable->sortItems($sortColumn, $request->get_param('sortDirection'));
+        }
```

Comment:
Wait, so is the sorting not occurring within the query? We can't sort after the query, otherwise what we're getting doesn't truly reflect the absolute sort order, only the relative sort order of the subset.

Reply:
I will update this so that sorting is done in DB.

Reply:
One idea: If we use the query builder, we can make it the responsibility of the column to know how to apply the sorting needs to be applied. I'm ok with their being constraints on this and simply saying that some (even many) columns simply aren't sortable. That's a reasonable constraint.

Reply:
We could pass the QB instance. Hmmm, good idea.